go.mod

```diff
@@ -12,8 +12,8 @@ require (
 	github.com/hashicorp/terraform-plugin-log v0.9.0
 	github.com/hashicorp/terraform-plugin-testing v1.4.0
 	github.com/jpillora/maplock v0.0.0-20160420012925-5c725ac6e22a
-	go.temporal.io/api v1.38.0
-	go.temporal.io/sdk v1.29.1
+	go.temporal.io/api v1.40.0
+	go.temporal.io/sdk v1.29.2-0.20241025174857-2bd372c351b2
```

Comment:
is this an rc? If so, should we wait on the released driver before including this version?

Reply:
ack - makes sense